triton-server/src/notification.ts

```diff
@@ -3,55 +3,87 @@ import nodemailer from "nodemailer"
 import * as email from "./contact-service"
 import { TritonDataset } from "./api/api-types"
 import { getFreezeManAuthenticatedAPI } from "./freezeman/api"
+import { defaultDatabaseActions } from "./download/actions"
+import { logger } from "./logger"
+import dayjs, { Dayjs } from "dayjs"
+import config from "../config"
+
+interface UpdatedTritonDataset
+    extends Omit<TritonDataset, "latest_release_update"> {
```

Comment:
omit variable from the extended interface to change its type?

Reply:
Indeed

Reply:
i unresolved this comment due to our face to face discussion, what would be the reasoning to have the omitted variable then newly integrated as a different variable type?

Reply:
so after discussing this topic, we found out that the latest_release_update variable from TritonDataset will never return undefined, this means that we should just change the base type of the latest_release_update variable from TritonDataset to not be undefined